rust/operator-binary/src/main.rs

```diff
@@ -58,21 +60,57 @@ async fn main() -> anyhow::Result<()> {
         Command::Run(ProductOperatorRun {
             product_config,
             watch_namespace,
-            tracing_target,
+            telemetry_arguments,
             cluster_info_opts,
         }) => {
-            stackable_operator::logging::initialize_logging(
-                "AIRFLOW_OPERATOR_LOG",
-                APP_NAME,
-                tracing_target,
-            );
-            stackable_operator::utils::print_startup_string(
-                crate_description!(),
-                crate_version!(),
-                built_info::GIT_VERSION,
-                built_info::TARGET,
-                built_info::BUILT_TIME_UTC,
-                built_info::RUSTC_VERSION,
+            let _tracing_guard = Tracing::builder()
+                // TODO (@Techassi): This should be a constant
+                .service_name("airflow-operator")
+                .with_console_output((
+                    // TODO (@Techassi): Change to CONSOLE_LOG, create constant
+                    "AIRFLOW_OPERATOR_LOG",
+                    LevelFilter::INFO,
+                    !telemetry_arguments.no_console_output,
+                ))
+                // note, before, log dir was set via an env: `AIRFLOW_OPERATOR_LOG_DIRECTORY`.
+                // See: https://github.com/stackabletech/operator-rs/blob/f035997fca85a54238c8de895389cc50b4d421e2/crates/stackable-operator/src/logging/mod.rs#L40
+                // Now it will be `ROLLING_LOGS` (or via `--rolling-logs <DIRECTORY>`)
+                .with_file_output(telemetry_arguments.rolling_logs.map(|log_directory| {
+                    let rotation_period = telemetry_arguments
+                        .rolling_logs_period
+                        .unwrap_or(RollingPeriod::Hourly)
+                        .deref()
+                        .clone();
+
+                    Settings::builder()
+                        // TODO (@Techassi): Change to CONSOLE_LOG or FILE_LOG, create constant
+                        .with_environment_variable("AIRFLOW_OPERATOR_LOG")
+                        .with_default_level(LevelFilter::INFO)
+                        .file_log_settings_builder(log_directory, "tracing-rs.log")
+                        .with_rotation_period(rotation_period)
+                        .build()
+                }))
+                .with_otlp_log_exporter((
+                    "OTLP_LOG",
+                    LevelFilter::DEBUG,
+                    telemetry_arguments.otlp_logs,
+                ))
+                .with_otlp_log_exporter((
+                    "OTLP_TRACE",
+                    LevelFilter::DEBUG,
+                    telemetry_arguments.otlp_logs,
+                ))
```

Comment:
Accidental copy/paste error:

```suggestion
                .with_otlp_trace_exporter((
                    "OTLP_TRACE",
                    LevelFilter::DEBUG,
                    args.common.telemetry_arguments.otlp_traces,
                ))
```